raml/api.raml

```diff
@@ -55,6 +55,18 @@ mediaType: application/json
               schema: !include schemas/404.json
               example: !include response_examples/404/region.json
 
+    /security/issues/{period}:
```

Comment:
Will you introduce /security/stats/{period} API endpoint in this pull request? Or this will be doe separately?